sktime/transformations/base.py

```diff
@@ -468,6 +469,10 @@ def fit(self, X, y=None):
             self._is_fitted = True
             return self
 
+        # if requires_y is set, y is required in fit and update
```

Comment:
is the comment correct or should change y by x ? Like: # if requires_x is set, x is required in fit and update

Reply:
This should apply for both x and y. btw, closed PR are not regularly monitored, so if you would like to bring sth to attentions of core devs, use discussions or issues. Activity in closed PR are spotted usually only by accident, as they may not trigger notifications.

Reply:
I see you already did here: https://github.com/sktime/sktime/issues/6687 - thanks!